jetty-util/src/main/java/org/eclipse/jetty/util/ArrayTrie.java

```diff
@@ -56,7 +56,8 @@
      * 32 to cover case insensitive alphabet and a few other common
      * characters.
      */
-    private static final int ROW_SIZE = 32;
+    private static final int ROW_SIZE = 48;
```

Comment:
It's certainly worth measuring the space & time difference between row sizes of 32 and 48.

Reply:
My gut feeling is that the extra space for digits means that we can now more often just allocate the normal table and avoid the secondary larger table more often.